### PR TITLE
Hassio show changelogs

### PR DESCRIPTION
**Pull request by c727**
- linked HA release notes
- show add-on changelog if available (currently as JS alert until we have a generic dialog component)
ddddd

dd

### CODE REVIEW

**Comment by c727** [comment]
maybe I should move `UPDATE` to the left side and remove `CHECK / OPEN` ?

File: hassio/dashboard/hassio-hass-update.html [hunk 1]
```diff
@@ -35,10 +36,14 @@
               </template>
             </div>
             <div class='card-actions'>
+              <a
+                href='https://github.com/home-assistant/home-assistant/releases'
```
Comment (pvizeli) [inline hassio/dashboard/hassio-hass-update.html]: maybe we should point to correct version

File: hassio/addon-view/hassio-addon-info.html [hunk 2]
```diff
@@ -235,6 +238,11 @@
     const data = { auto_update: !this.addon.auto_update };
     this.hass.callApi('POST', `hassio/addons/${this.addonSlug}/options`, data);
   }
+
+  openChangelog() {
+    this.hass.callApi('get', `hassio/addons/${this.addonSlug}/changelog`)
+      .then(data => alert(data));
```
Comment (pvizeli) [inline hassio/addon-view/hassio-addon-info.html]: instead a js alert, we can show a UI like snapshot and show the markdown inside this?

File: hassio/dashboard/hassio-hass-update.html (hunk 1 above)
Comment (balloob) [reply hassio/dashboard/hassio-hass-update.html]: Prefer to just point at least at https://home-assistant.io/blog/ , preferable point at the right url.

Comment (c727) [reply hassio/dashboard/hassio-hass-update.html]: GH release note list all breaking changes, even if you skipped some releases. Blog url has to be fetched from somewhere

**Comment by c727** [comment]
can someone help with the dialog/backdrop please? :(

(I want to use this popup also for the long_description later)

**Review by balloob — APPROVED** [review]

**Comment by c727** [comment]
^the popup is still a bit broken but useable :P

**Pull request merged**